exchanges/kucoin/kucoin_wrapper.go

```diff
@@ -0,0 +1,1480 @@
+func (ku *Kucoin) WithdrawCryptocurrencyFunds(ctx context.Context, withdrawRequest *withdraw.Request) (*withdraw.ExchangeResponse, error) {
+	if err := withdrawRequest.Validate(); err != nil {
+		return nil, err
+	}
+	withdrawalID, err := ku.ApplyWithdrawal(ctx, withdrawRequest.Currency.String(), withdrawRequest.Crypto.Address, withdrawRequest.Crypto.AddressTag, withdrawRequest.Description, withdrawRequest.Crypto.Chain, "INTERNAL", false, withdrawRequest.Amount)
+	if err != nil {
+		return nil, err
+	}
+	return &withdraw.ExchangeResponse{
+		ID: withdrawalID,
+	}, nil
+}
+
+// WithdrawFiatFunds returns a withdrawal ID when a withdrawal is submitted
+func (ku *Kucoin) WithdrawFiatFunds(_ context.Context, _ *withdraw.Request) (*withdraw.ExchangeResponse, error) {
+	return nil, common.ErrFunctionNotSupported
+}
+
+// WithdrawFiatFundsToInternationalBank returns a withdrawal ID when a withdrawal is
+// submitted
+func (ku *Kucoin) WithdrawFiatFundsToInternationalBank(_ context.Context, _ *withdraw.Request) (*withdraw.ExchangeResponse, error) {
+	return nil, common.ErrFunctionNotSupported
+}
+
+func orderTypeToString(oType order.Type) (string, error) {
+	switch oType {
+	case order.Limit:
+		return "limit", nil
+	case order.Market:
+		return "market", nil
+	case order.StopLimit:
+		return "limit_stop", nil
+	case order.StopMarket:
+		return "market_stop", nil
+	case order.AnyType, order.UnknownType:
+		return "", nil
+	default:
+		return "", order.ErrUnsupportedOrderType
+	}
+}
+
+// GetActiveOrders retrieves any orders that are active/open
+func (ku *Kucoin) GetActiveOrders(ctx context.Context, getOrdersRequest *order.MultiOrderRequest) (order.FilteredOrders, error) {
+	if getOrdersRequest == nil {
+		return nil, common.ErrNilPointer
+	}
+	err := ku.CurrencyPairs.IsAssetEnabled(getOrdersRequest.AssetType)
+	if err != nil {
+		return nil, err
+	}
+	if getOrdersRequest.Validate() != nil {
+		return nil, err
+	}
+	format, err := ku.GetPairFormat(getOrdersRequest.AssetType, true)
+	if err != nil {
+		return nil, err
+	}
+	pair := ""
+	orders := []order.Detail{}
+	switch getOrdersRequest.AssetType {
+	case asset.Futures:
+		if len(getOrdersRequest.Pairs) == 1 {
+			pair = format.Format(getOrdersRequest.Pairs[0])
+		}
+		sideString, err := ku.orderSideString(getOrdersRequest.Side)
+		if err != nil {
+			return nil, err
+		}
+		oType, err := orderTypeToString(getOrdersRequest.Type)
+		if err != nil {
+			return nil, err
+		}
+		futuresOrders, err := ku.GetFuturesOrders(ctx, "active", pair, sideString, oType, getOrdersRequest.StartTime, getOrdersRequest.EndTime)
+		if err != nil {
+			return nil, err
+		}
+		var enabledPairs currency.Pairs
+		enabledPairs, err = ku.GetEnabledPairs(asset.Futures)
+		if err != nil {
+			return nil, err
+		}
+		for x := range futuresOrders.Items {
+			if !futuresOrders.Items[x].IsActive {
+				continue
+			}
+			dPair, err := enabledPairs.DeriveFrom(futuresOrders.Items[x].Symbol)
+			if err != nil {
+				return nil, err
+			}
```

Comment:
```suggestion
if err !=nil {
    if errors.Is(err, currency.ErrPairNotFound) {
        continue
    }
    return nil, err
}
```

Just a bit nicer for the user